src/main/java/Duke.java

```diff
@@ -1,10 +1,99 @@
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        welcomeMessage();
+
+        Scanner sc = new Scanner(System.in);
+        List<Task> tasks = new ArrayList<Task>();
+        Boolean exitProgram = false;
```

Comment:
Would it be better to be called isExitCommandUsed instead of exitProgram as a boolean variable?